masterlist.yaml

```
@@ -12788,7 +12800,9 @@ plugins:
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/8683/' ]
     after: [ 'Landscape Fixes For Grass Mods.esp' ]
     inc: [ 'DragonBridgeBoardwalk.esp' ]
-    tag: [ Actors.AIPackages ]
+    tag:
+      - Actors.AIPackages
+      - -Invent.Remove
```

Same suggestion as the Invent for Ars Metallica.

```suggestion
      - -Invent
      - Invent.Add
```